requirements/common.txt

```diff
@@ -55,3 +55,5 @@ diff-match-patch
 blinker
```

Comment:
blinker is used, let it be there for now, I've removed other unused dependencies and also fixed all pep8 violations